server/serverUtils/response.js

```diff
@@ -29,16 +33,24 @@ const sendErr = (res, err) => {
 }
 
 const sendFile = (res, file) => {
-  res.setHeader('Content-Disposition', `attachment; filename=${File.getName(file)}`)
-  res.setHeader('Content-Length', File.getSize(file))
-  // res.set('Content-Type', 'text/csv')
-
+  sendFilePrepare(res, File.getName(file), File.getSize(file))
   res.write(file.content, 'binary')
   res.end(null, 'binary')
 }
 
+const sendFilePrepare = (res, fileName, fileSize = null, contentType = null) => {
```

Comment:
I would call this function setFileOutput or similar?

Reply:
it's a function that modify a response, setting the content type and disposition... it's more something like "prepare to file write"

Reply:
Yes you're setting whatever the response needs to output a file. I was commenting about what is easier to understand when you read ```Response.sendFilePrepare(...)``` or ```Response.setContentTypeFile(...)``` or better name... What do you think?